chat/chat.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/anyurl', (req,res) =>{
```

Comment:
make url more reasonable. like... `/newChat`

Reply:
hey song i changed and pushed the new files. Please check it